packages/shopping-cart/src/types.ts

```diff
@@ -211,6 +211,7 @@ export interface RequestCartProduct {
 	volume: number;
 	quantity: number | null;
 	extra: RequestCartProductExtra;
+	is_domain_registration: boolean;
```

Comment:
You may be right, but I haven't found a location in the shopping-cart endpoint where this property is used. Do you know of one? I was pretty sure that this property was dynamically set based on the product ID. After all, shouldn't the shopping-cart endpoint know if a product is a domain registration or not?

If this is actually needed, is it possible to make this change on the backend instead inside the endpoint? If a product is a domain registration, always set `is_domain_registration` to true (which is what I thought was already the case)?

Reply:
Here's an example of adding a domain product to the cart without this PR. This is the request payload (note that there is no `is_domain_registration` set):

<img width="347" alt="Screen Shot 2022-01-19 at 12 12 05 PM" src="https://user-images.githubusercontent.com/2036909/150180421-679c7b4b-3e64-495d-bfe2-719dcc846305.png">

And here's the response (note that `is_domain_registration` is set to true):

<img width="293" alt="Screen Shot 2022-01-19 at 12 12 18 PM" src="https://user-images.githubusercontent.com/2036909/150180509-1ecd4a3e-03d1-40d2-8eea-9d9837d02433.png">

Reply:
> If this is actually needed, is it possible to make this change on the backend instead inside the endpoint?

I don't think adding it in the endpoint helps fix this bug, but I could be wrong. I've updated the PR description so hopefully it helps clarify the intent behind this fix. When the user goes from the domains step to plans step of the signup flow, there is no shopping cart endpoint request made yet and there is also no site created yet, so looks to me that this should be a front-end fix. There might be a better solution that I'm not seeing.